tests/testthat/test-derive_var_afeye.R

```diff
@@ -96,15 +96,15 @@ test_that("derive_var_afeye Test 4: Deprecation of lat_vals", {
     "XXX001", "P01", "RIGHT", "", "", NA_character_,
   )
 
-  expect_warning(
+  expect_error(
     actual_output4 <- expected_output4 %>%
       select(-AFEYE) %>%
       derive_var_afeye(
         loc_var = AELOC,
         lat_var = AELAT,
         lat_vals = c("LEFT", "RIGHT", "BILATERAL")
       ),
-    class = "lifecycle_warning_deprecated"
+    class = "lifecycle_error_deprecated"
   )
 
   expect_dfs_equal(
```

Comment:
Same as above comment.

Reply:
Thank you for your suggestion. I have updated it.